modules/internal/ExternalArray.chpl

```diff
@@ -180,6 +180,7 @@ module ExternalArray {
     const dom;
     
     const _ArrInstance: chpl_external_array;
+    const elts = _ArrInstance.elts: _ddata(eltType);
```

Comment:
For some reason I expected this to be set in the initializer rather than here. Should work either way, I'm fine with this.

Reply:
elts gets handled via omitted initialization in the initializer :)